src/main/java/org/cactoos/io/TeeInput.java

```diff
@@ -45,6 +45,9 @@
  * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
  * @since 0.1
+ * @todo #631 This class needs more test cases. Currently, only a set of
```

Comment:
@proshin-roman second and third line of this puzzle should start with a space. Also, add the estimation:

```
@todo #631:30min This class needs more test cases. Currently, only a set of
 ctors is covered by tests: ctors which use Bytes and byte array as an
 input. All other ctors should be covered too.
```

Reply:
@amihaiemil I've both new comments. Btw: Qulice usually checks indentation for JavaDocs, but skips these two problems - that's a bit strange :)